bild.py

```diff
@@ -42,6 +43,8 @@
 PYTHON3_TARGET	= "../antlr4-python3"
 CSHARP_TARGET	= "../antlr4-csharp"
 
+JAVA_VERSION = os.environ.get('JAVA_VERSION', None)
```

Comment:
this is fixed now. e.g., output in bild.log is:

[01/10/15 21:34:59 javac ./bild.py:127 bilder.py:452] javac -version
[01/10/15 21:35:00 javac ./bild.py:127 bilder.py:452] javac 1.7.0_25
[01/10/15 21:35:00 javac ./bild.py:127 bilder.py:454] javac -d /Users/parrt/antlr/code/antlr4/out/runtime -cp /Users/parrt/antlr/code/antlr4/out/runtime:runtime/Java/lib/org.abego.treelayout.core.jar -source 1.6 -target 1.6 -nowarn -Xlint -Xlint:-serial -g -sourcepath gen4:runtime/JavaAnnotations/src:runtime/Java/src /Users/parrt/antlr/code/antlr4/gen4/org/antlr/v4/runtime/tree/xpath/XPathLexer.java

On Jan 10, 2015, at 8:50 PM, ericvergnaud <notifications@github.com> wrote:

> In bild.py:
> 
> > @@ -42,6 +43,8 @@
> >  PYTHON3_TARGET	= "../antlr4-python3"
> >  CSHARP_TARGET	= "../antlr4-csharp"
> >  
> > +JAVA_VERSION = os.environ.get('JAVA_VERSION', None)
> We have no need for java 1.7 right now, although we should start advertising that next major release will move to 1.7
> …
> —
> Reply to this email directly or view it on GitHub.
>